torch_xla/csrc/helpers.h

```diff
@@ -324,8 +324,14 @@ class XlaHelpers {
 
   template <typename T>
   static xla::Literal Range(T start, T end, T step) {
+    int64_t range_size = (int64_t)std::ceil((end - start) / step);
+    xla::Array<T> range({range_size});
+    range.FillWithMultiples(step);
+    range.Each([start](absl::Span<const int64_t> index, T* value) {
+      *value += start;
+    });
     return xla::LiteralUtil::CreateR1<T>(
-        runtime::util::Range<T>(start, end, step));
+        absl::MakeConstSpan(range.data(), range.dim(0)));
```

Comment:
Can you create from the array directly? https://github.com/openxla/xla/blob/fbcb441c8b5a327c7092bc21d31a3e598f11dd0c/xla/literal_util.h#L132-L133

Reply:
Nice find! I missed that API.